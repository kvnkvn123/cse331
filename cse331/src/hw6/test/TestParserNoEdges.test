# Test the parser on a graph that contains several with no edges

LoadGraph simple simpleGraph0.tsv
ListNodes simple

LoadGraph simple2 simpleGraph2.tsv
ListNodes simple2
ListChildren simple2 aaa
ListChildren simple2 bbb
ListChildren simple2 ccc
ListChildren simple2 ddd