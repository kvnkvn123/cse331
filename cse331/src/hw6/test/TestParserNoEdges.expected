# Test the parser on a graph that contains several with no edges

loaded graph simple
simple contains:

loaded graph simple2
simple2 contains: aaa bbb ccc ddd
the children of aaa in simple2 are:
the children of bbb in simple2 are:
the children of ccc in simple2 are:
the children of ddd in simple2 are: